### ISSUE TITLE
Some users need to install Microsoft Visual C++ Redistibutable

### ISSUE THREAD

**Issue by enoearias** [LabVIEWCommunityTraining/cti-virtual-machine-documentation#8]
So, I'm working with an user and Virtual Box complains about Microsoft Visual C++

![WhatsApp Image 2024-01-31 at 1 54 56 PM](https://github.com/LabVIEWCommunityTraining/cti-virtual-machine-documentation/assets/5545396/158f2eb7-b765-4780-a1a3-2c04ac14c4c0)

I searched for this under microsoft website and it fixes the issue, I'm writting this issue just as documentation and maybe adding a side note if someone else encounters this situation.

[Microsoft Site](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170)

**Comment by gt-thetic** [comment]
We have moved to VirtualBox 7.1.6 for creating the VM, so this may no longer be an issue. I will close it as expired.  If it comes up again, we'll have to reopen it.  OK with you @enoearias ?

**Comment by enoearias** [comment]
Yes, the last time I installed virtual box everything worked just fine.

Thank you!
